sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -201,7 +203,18 @@ def list_configuration_settings(
         """
 
     @distributed_trace
-    def list_configuration_settings(self, **kwargs) -> ItemPaged[ConfigurationSetting]:
+    def list_configuration_settings(self, *args, **kwargs) -> ItemPaged[ConfigurationSetting]:
+        accept_datetime = kwargs.pop("accept_datetime", None)
+        if isinstance(accept_datetime, datetime):
+            accept_datetime = str(accept_datetime)
```

Comment:
Is a certain format expected for the string date?

Reply:
I don't think it has format expectations, I checked the generated API surface doesn't mention it.